docs/defects-api.yml

```diff
@@ -324,7 +295,10 @@ components:
         inspectionTypes:
           type: array
           items:
-            $ref: "#/components/schemas/inspectionType"
+            type: string
+            enum:
+              - basic
+              - normal
 
     vehicleType:
```

Comment:
I don't think we are using vehicle type any more, can this be removed?

I think it is also referenced on the `ivaDefect` type and needs to be removed from there.